ci/conda_env_unix.yml

```diff
@@ -19,6 +19,8 @@
 
 autoconf
 ccache
+google-cloud-cpp=1.20.0
```

Comment:
What is the reason for the exact pin here?

Reply:
The currently available version in conda (1.24) is incompatible with the way I initially packaged it. I believe I saw in the CI run that `storage_client` was missing or something, as the google-cloud-cpp package has [started namespacing its exports](https://github.com/googleapis/google-cloud-cpp/blob/5bf147226e9333c5106d5fc8b8bba62657eeab97/google/cloud/storage/legacy.cmake.in#L28) (finally!).

Given the hard link against abseil libraries (as gRPC does too), I decided that the version was safer pinned.

I'm hoping to upgrade the dependency when there is code using it (which would be the GCS implementation of the Arrow file interfaces).

Reply:
I determined required abseil libraries manually. CI error helped me.